testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -224,13 +222,9 @@
 XTC_sub_sol = resource_filename(__name__, 'data/cobrotoxin.xtc')
 PDB_sub_sol = resource_filename(__name__, 'data/cobrotoxin.pdb')
 PDB_xlserial = resource_filename(__name__, 'data/xl_serial.pdb')
-XTC_single_frame = resource_filename(
```

Comment:
yes I noticed they weren't needed in the first place. They test nothing that isn't tested in the other files